crowdin.yml

```diff
@@ -0,0 +1,28 @@
+# To use the crowdin-cli, set an access token in CROWDIN_PERSONAL_TOKEN or a ~/.crowdin.yml
```

Comment:
Does this need me to add secrets to the organization or is this on Crowdin's end?

Reply:
This is if a developer wants to the the CLI. The github integration does its own thing, not sure what it'll use exactly but we don't need to do anything more for that. It might use my account since I configured it?